web-ui/src/login/login.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html xmlns:t="http://twistedmatrix.com/ns/twisted.web.template/0.1">
+  <head>
+    <link rel="icon" type="image/png" href="/assets/images/Favicon.png" />
+    <meta charset="utf-8"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+    <meta name="viewport" content="user-scalable=no, width=device-width, initial-scale=1.0, maximum-scale=1.0"/>
+    <title>Pixelated - Login</title>
+    <link rel="stylesheet" type="text/css" href="/assets/normalize.min.css" />
+    <link rel="stylesheet" type="text/css" href="/assets/login.css" />
+    <link rel="stylesheet" type="text/css" href="/assets/opensans.css" />
+  </head>
+  <body>
+    <div class="content">
+      <div class="login">
+        <img class="logo" src="/assets/images/logo-orange.svg" alt="Pixelated logo"/>
+        <p t:render="error_msg" class="error"></p>
+        <div id="root"/>
```

Comment:
Because the next line (`<p t:render="error_msg" class="error"></p>`) is a template, which react doesn't understand. If we have time, we'll try to fix it in this issue.